Woof/Woof/View/OwnerMainTabView.swift

```diff
@@ -0,0 +1,72 @@
+import SwiftUI
+
+struct OwnerMainTabView: View {
+    @State private var selection: Tab = .sitters
+
+    init() {
+        customizeTabBar()
+    }
+
+    var body: some View {
+        NavigationView {
+            TabView(selection: $selection) {
+                Group {
+                    Text("List of pet sitters")
+                        .tabItem {
+                            Label("Sitters", systemImage: "magnifyingglass")
```

Comment:
Please make images type-safe and less error-prone.
I'd like to see smth like this:
```swift
Label("Sitters", systemImage: Image.SittersTabIconName)
```

Reply:
Add extension for Image with icon names + add tests.